CONTRIBUTING.md

```diff
@@ -28,6 +26,8 @@ You can find them by using this query (check the link) for [newbie friendly issu
 The Jenkins core build flow is built around Maven.
 You can read a description of the [building and debugging process here](https://www.jenkins.io/doc/developer/building/).
 
+### Building the WAR file
```

Comment:
Adding a subheading for structural purposes so that I can link this section from its consumers (of which there are now two rather than one) without the vagueness of the previous wording ("after the above command") which would now be ambiguous now that there are two consumers of this section.